src/PimApi/Model/ProductValue.php

```diff
@@ -6,10 +6,13 @@
 
 class ProductValue
 {
+    /**
+     * @param string|bool|int|float|array<string>|null $value
+     */
```

Comment:
1. because of the new `array` type that is not enough precise in pure PHP for our static analysis tools
2. as I remember, phpstan forced me this order, I will try to be consistent